src/main/java/ro/msg/learning/shop/controller/ProductController.java

```diff
@@ -0,0 +1,57 @@
+package ro.msg.learning.shop.controller;
+
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import ro.msg.learning.shop.dto.ProductDto;
+import ro.msg.learning.shop.service.IProductService;
+import ro.msg.learning.shop.utils.mappers.ProductMapper;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@RestController
+@RequiredArgsConstructor
+public class ProductController {

```

Comment:
You should add the @RequestMapping annotation here. You can add the value of your '/products' path also to not repeat yourself over every request mapping afterwards. The same applies to the other controller too.